migration/migrator/data/submitty_db.sql

```diff
@@ -663,6 +663,13 @@ ALTER TABLE ONLY public.vcs_auth_tokens
     ADD CONSTRAINT vcs_auth_tokens_pkey PRIMARY KEY (id);
 
 
+--
+-- Name: sessions_session_expires_idx; Type: INDEX; Schema: public; Owner: -
+--
+
+CREATE INDEX sessions_session_expires_idx ON public.sessions USING btree (session_expires);
```

Comment:
Good catch.  The index should be removed.  This index would have improved the original situation substantially, given that delete operations were being executed on every request, but I think it isn't helpful here.  That was the first thing I added, but it should be removed since it's now irrelevant.